infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -1064,13 +1061,7 @@ public void append(List<LogData> range) {
 
         // Extract all addresses associated with the provided write range.
         Set<Long> pendingWrites = range.stream()
-                .map(entry -> entry.getGlobalAddress()).collect(Collectors.toSet());
```

Comment:
Why remove the check?

Reply:
Because the verify() method already did the check for uniqueness.